app/code/core/Mage/Core/etc/system.xml

```diff
@@ -1679,5 +1679,526 @@
+                            <frontend_type>multiselect</frontend_type>
+                            <source_model>adminhtml/system_config_source_grid_catalog_product_column</source_model>
+                            <sort_order>4</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>0</show_in_website>
+                            <show_in_store>0</show_in_store>
+                            <can_be_empty>1</can_be_empty>
+                            <depends>
+                                <enabled>1</enabled>
+                            </depends>
+                        </columns>
+                        <media_image translate="label">
+                            <label>Product Image Preview</label>
+                            <comment><![CDATA[Select product media image types]]></comment>
+                            <frontend_type>multiselect</frontend_type>
+                            <source_model>adminhtml/system_config_source_grid_catalog_product_mediaImage</source_model>
+                            <sort_order>5</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>0</show_in_website>
+                            <show_in_store>0</show_in_store>
+                            <can_be_empty>1</can_be_empty>
+                            <depends>
+                                <enabled>1</enabled>
+                            </depends>
+                        </media_image>
+                        <media_image_width translate="label comment">
+                            <label>Product Image Width</label>
+                            <comment><![CDATA[Set product image preview width]]></comment>
+                            <validate>validate-number validate-zero-or-greater</validate>
+                            <sort_order>51</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>0</show_in_website>
+                            <show_in_store>0</show_in_store>
+                            <depends>
+                                <enabled>1</enabled>
+                                <!-- <media_image>1</media_image> -->
+                            </depends>
+                        </media_image_width>
+                    </fields>
+                </related_product_grid>
+                <up_sell_product_grid>
+                    <label>Catalog > Manage Products > Edit | Up-Sells</label>
+                    <sort_order>3</sort_order>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>0</show_in_website>
+                    <show_in_store>0</show_in_store>
+                    <fields>
+                        <enabled>
+                            <label>Enable</label>
```

Comment:
All **Enable** words must be replaced with **Enabled** to follow the Backend rule related to activate a feature with two options Yes/No.